master/buildbot/test/util/misc.py

```diff
@@ -89,6 +92,11 @@ def deferToThread(f, *args, **kwargs):
         self.addCleanup(_setReactor, None)
         self.addCleanup(self.reactor.stop)
 
+        if use_asyncio:
+            self.loop = TwistedLoop(self.reactor)
```

Comment:
I would prefer if we used a less generic name. In the case of reactor it's obvious, whole Buildbot is created using Twisted. `loop` can mean anything on the other hand. Anything like `asyncio_loop`, `event_loop` or something like that would be fine.